File: NGOISAO4.py
'''
Viết chương trình in ra màn hình tam giác sao cân với chiều cao h nhập vào từ bàn phím.

Ví dụ 1:

Input	Output
3
__*__
_***_
*****

Ví dụ 2:

Input	Output
5
____*____
___***___
__*****__
_*******_
*********

Ví dụ 3:

Input	Output
6
_____*_____
____***____
___*****___
__*******__
_*********_
***********

Lưu ý: 
Xuất chính xác từng ký tự. Không xuất dư hoặc thiếu khoảng trắng, không xuất dư endl dòng cuối cùng.
Các ký tự gạch nối màu đen (_) ở phía trên đại diện cho khoảng trắng mà các bạn phải xuất ra màn hình. Phải có các khoảng trắng này thì mới được tính là tam giác cân.
'''
h = int(input())
for i in range(h):
    spaces = ' ' * (h - i - 1)
    stars = '*' * (2 * i + 1)
    line = spaces + stars + spaces
    if i != h - 1:
        print(line)
    else:
        print(line, end="")

File: LTCB/C++/VongLap/NGOISAO4.cpp
// Viết chương trình in ra màn hình tam giác sao cân với chiều cao h nhập vào từ bàn phím.
#include <iostream>
using namespace std;

#define sn int
#define nhap cin
#define xuat cout
#define kt return 0

sn main(){
    sn n;
    nhap >> n;
    
    for(sn i = 0; i < n; i++){
        for(sn j = 1; j < 2*n; j++){
            if(j < n - i || j > n+i) xuat <<" ";
            else xuat <<"*";
        }
        if(i != n -1) xuat << endl;
    }
    kt;
}